docs/MS Excel/Формулы/Вычисление дат.md

```diff
@@ -0,0 +1,64 @@
+# Вычисление дат 
+
+## Сложить или вычесть дни из даты
+
+1. Введите даты сходимой даты в столбце A.
+2. Введите количество дней, в течение дней в которые нужно добавить или вычесть столбец B. Вы можете ввести отрицательное число, чтобы вычесть дни из даты начала, и положительное число, чтобы прибавить к дате.
+3. В ячейке C2 введите **=A2+B2** и скопируйте его при необходимости.
+![](assets/17036199978746.png)
+
+
+## Сложить или вычесть месяцы из даты с помощью функции ДАТАМЕСТ
+
+Для **функции** ДАТАМЕСТ требуются два аргумента: дата начала и количество месяцев, которые нужно прибавить или вычесть. Чтобы вычесть месяцы, введите отрицательное число в качестве второго аргумента. Например, =ВРЕМЯ("15.09.19";-5) возвращает 15.04.19.
+
+![](assets/17036200371019.png)
```

Comment:
[ch-md-error-image-alt-text](https://chaberch.github.io/how-to-chaberch/markdown/#ch-md-error-image-alt-text)